.env.example

```diff
@@ -13,3 +13,4 @@ REGISTRY_ACCOUNT_CREATE_URL = "https://dev.bcregistry.ca/business/auth/choose-au
 KEYCLOAK_CONFIG_PATH = "keycloak-dev.json"
 AUTH_API_URL = "https://auth-api-dev.apps.silver.devops.gov.bc.ca/api/v1/"
 STATUS_API_URL = "https://status-api-dev.apps.silver.devops.gov.bc.ca/api/v1/"
+SBC_AUTH_LD_CLIENT_ID = "5db9da115f58e008123cd783"
```

Comment:
@doug-lovett This needs to be added to all environments.

Dev = 5db9da115f58e008123cd783
Test = 5db9d3c4076e7508aaf7db1d
Prod = 5db9d3c4076e7508aaf7db1e